frontend/src/components/common/SectionFilterHeader.tsx

```diff
@@ -21,10 +23,13 @@ interface SectionFilterHeaderProps extends SectionHeaderProps {
 
 export default function SectionFilterHeader(props: SectionFilterHeaderProps) {
   const { noNamespaceFilter = false, noSearch = false, ...headerProps } = props;
+  const location = useLocation();
+  const namespaceInURL = getFilterValueByNameFromURL('namespace', location, true);
   const filter = useTypedSelector(state => state.filter);
   const dispatch = useDispatch();
 
-  const hasNamespaceFilters = !noNamespaceFilter && filter.namespaces.size > 0;
+  const hasNamespaceFilters = namespaceInURL.length !== 0 ||
+  (!noNamespaceFilter && filter.namespaces.size > 0);
```

Comment:
This logic will activate the filter header even when there's no namespace filter to be shown (some of the views don't have a namespace to be used).